packages/client/src/forms/ProfileChangePassword/index.tsx

```diff
@@ -66,6 +71,7 @@ export const ProfileChangePasswordForm = () => {
 			{({isSubmitting}) => (
 				<Form className='form'>
 					<Avatar
+						src={initialValues.avatar}
```

Comment:
Здесь можно обратиться напрямую к переменной `user`
```suggestion
						src={user?.avatar ?? null}
```